app/views/articles/edit.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Edit Article</h1>
+
+<%= render "form", article:@article %>
```

Comment:
What does this line mean?

Reply:
Is the view for Edit Article, so this view will use and render the `_form.html.erb` template and send the variable  `article` which is an internal variable in form template and uses the instance variable from __ArticlesController__.